common/scripts/gather_packages.py

```diff
@@ -121,27 +121,30 @@ def determineDistTag(branchName, currentVer, latestVer, previousVer):
     packagesToPublish = True
     print ("The package does not yet exist.  Copying " + packageName + " to staging area.")
     shutil.copy(artifact, stagingDir)
-    continue
 
   if 0 != len(serverVer):
     print ("The version already exists.  Skipping...")
     continue
 
   packagesToPublish = True
   print ("Local version is newer than on the server.  Copying package " + packageName + " to staging area.")
```

Comment:
Can we pick one of the two messages? We'll print this one and this print ("The package does not yet exist.  Copying " + packageName + " to staging area.") now, right?

Reply:
chose to use the "local version is newer" message